backend/tests/test_exporter.py

```diff
@@ -0,0 +1,61 @@
+import tempfile
+
+import openzim_uploader
+import pytest
+
+from backend.constants import BackendConf
+from backend.exporters import ExporterInterface
+from backend.exporters.kiwix_public_exporter import KiwixPublicExporter
+
+
+@pytest.mark.asyncio
+async def test_exporter_interface():
+
+    with pytest.raises(NotImplementedError):
+        for _ in await ExporterInterface().export():
+            ...
+
+
+@pytest.mark.asyncio
+async def test_incorrect_path(monkeypatch):
+    def mock_named_temporary_file(delete=False):
+        class MockNamedTemporaryFile(object):
+            def __init__(self, *args, **kwargs):
+                self.name = "/fakefile"
+
+        return MockNamedTemporaryFile()
+
+    with monkeypatch.context() as mp:
+        mp.setattr(tempfile, "NamedTemporaryFile", mock_named_temporary_file)
+        assert await KiwixPublicExporter.export() == 0
```

Comment:
Misleading `== 0` while this should return `True` or `False`. This means it failed

Reply:
OK